src/couchapp/ddocs.rst

```diff
@@ -93,23 +92,23 @@ Reduce and rereduce functions
     :return: Reduces `values`
 
 Reduce functions takes two required arguments of keys and values lists - the
-result of the related map function - and optional third one which indicates if
-`rereduce` mode is active or not. `Rereduce` is using for additional reduce
+result of the related map function - and an optional third value which indicates
+if `rereduce` mode is active or not. `Rereduce` is used for additional reduce
 values list, so when it is ``true`` there is no information about related `keys`
 (first argument is ``null``).
 
-Note, that if produced result by `reduce` function is longer than initial
+Note that if the result of a `reduce` function is longer than the initial
 values list then a Query Server error will be raised. However, this behavior
-could be disabled by setting ``reduce_limit`` config option to ``false``:
+can be disabled by setting ``reduce_limit`` config option to ``false``:
 
 .. code-block:: ini
 
     [query_server_config]
     reduce_limit = false
 
-While disabling ``reduce_limit`` might be useful for debug proposes, remember,
-that main task of reduce functions is to *reduce* mapped result, not to make it
-even bigger. Generally, your reduce function should converge rapidly to a single
+While disabling ``reduce_limit`` might be useful for debug proposes, remember
+that the main task of reduce functions is to *reduce* mapped result, not to make
+them bigger. Generally, your reduce function should converge rapidly to a single
```

Comment:
It should probably say "to reduce mapped results" (which is probably what I read), or "to reduce the mapped result", in which case the singular "it" would be correct.